app/scripts/modules/azure/loadBalancer/loadBalancer.transformer.js

```diff
@@ -62,6 +62,9 @@ module.exports = angular
           }
 
           toEdit.probes = elb.probes;
+          if (elb.dnsName && elb.dnsName != 'dns-not-found') {
```

Comment:
can this use strict inequality instead? `!==`

Reply:
Sure. Updated code.

Reply:
Sure. Updated code.